gobblin-cluster/src/main/java/org/apache/gobblin/cluster/SingleTaskRunnerBuilder.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gobblin.cluster;
+
+class SingleTaskRunnerBuilder {
```

Comment:
nit: this is doable via Lombok builder as at other places within Gobblin, would help reduce some boilerplate code

Reply:
Thanks for the info. I thought about it. 
I created this builder class using intellij refactoring tool. So it doesn't take more effort than using Lombok. And the code is more explicit and less magical. e.g. code navigation works better with this version. I would like to optimize the code for reading not writing. 
I would prefer keeping this version.
Let me know if you disagree strongly. 
We can chat in person if you would like.